SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/RegisterAutoDestroyActorsTest/RegisterAutoDestroyActorsTest.cpp

```diff
@@ -26,17 +26,45 @@ void ARegisterAutoDestroyActorsTestPart1::BeginPlay()
 			// and spawn them in that radius
 			FVector SpawnPosition = FVector(200.0f, -200.0f, 0.0f);
 			FRotator SpawnPositionRotator = FRotator(0.0f, 360.0f / NumVirtualWorkers, 0.0f);
-			for(int i = 0; i != NumVirtualWorkers; ++i)
+			for (int i = 0; i != NumVirtualWorkers; ++i)
 			{
 				ACharacter* Character = World->SpawnActor<ACharacter>(SpawnPosition, FRotator::ZeroRotator);
-				NetTest->AssertTrue(IsValid(Character), FString::Printf(TEXT("Spawned ACharacter in worker %s"), *NetTest->GetFlowController(ESpatialFunctionalTestFlowControllerType::Server, i+1)->GetDisplayName()));
+				NetTest->AssertTrue(IsValid(Character), FString::Printf(TEXT("Spawned ACharacter %s in worker %s"), *(Character->GetName()), *NetTest->GetFlowController(ESpatialFunctionalTestFlowControllerType::Server, i + 1)->GetDisplayName()));
+				//UE_LOG(LogTemp, Warning, TEXT("Spawned ACharacter %s in worker %s"), *(Character->GetName()), *NetTest->GetFlowController(ESpatialFunctionalTestFlowControllerType::Server, i + 1)->GetDisplayName());
 				SpawnPosition = SpawnPositionRotator.RotateVector(SpawnPosition);
 			}
+			
+
 			NetTest->FinishStep();
 		});
+
+		
+	}
+
+	{  // Step 2 - Wait 5 seconds allow characters to transition to new workers
+
+		// Setup timer for next step
+		double endTime; 
+		bool startedTimer = false;
+
+	   AddServerStep(TEXT("SERVER__1_Wait_5_Seconds"), 1, [startedTimer, endTime](ASpatialFunctionalTest* NetTest) mutable -> bool {
+		   if (!startedTimer)
+		    {
+			   endTime = FPlatformTime::Seconds() + 5;
+			   startedTimer = true;
+		    }
+		    double currentTime = FPlatformTime::Seconds();
+			
+			return (currentTime > endTime);
+			},
+
+			[](ASpatialFunctionalTest* NetTest) {
+				NetTest->FinishStep();
+			}, nullptr, 6.0f);
```

Comment:
This is not a good practice to leave in, it makes the test take a lot of time to execute. I'd recommend reverting the Test to how it was before, since I don't understand why the current changes occurred (besides that we atm have this "bug" that you can only register actors to delete by their authority, which shouldn't be the case)

Reply:
Removed as requested.